ballerina/tests/tests.bal

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/os;
+import ballerina/test;
+
+configurable boolean isTestOnLiveServer = os:getEnv("IS_TEST_ON_LIVE_SERVER") == "true";
+configurable string apiVersion = "2025";
+configurable string accessToken = "mock-access-token";
+configurable string shopifyLiveServer = "mock-access-token";
+
+string serviceUrl = isTestOnLiveServer ? shopifyLiveServer : "http://localhost:8080/shopify";
+Client shopify = check new (
+    accessToken,
+    apiVersion,
+    {
+        followRedirects: {
+            enabled: true
```

Comment:
Any specific reason to enabling this by default?

Reply:
Shopify allows users to set custom domains for their stores which redirect to the original domain. To support this, it would be better if we allow this by default.